awkward-cpp/include/awkward/common.h

```diff
@@ -49,44 +52,43 @@
 #include <cstring>
 
 extern "C" {
-  struct EXPORT_SYMBOL Error {
-    const char* str;
-    const char* filename;
-    int64_t identity;
-    int64_t attempt;
-  };
+struct EXPORT_SYMBOL Error {
+  const char* str;
+  const char* filename;
+  int64_t identity;
+  int64_t attempt;
+};
 
-  const int8_t   kMaxInt8   =                 127;   // 2**7  - 1
-  const uint8_t  kMaxUInt8  =                 255;   // 2**8  - 1
-  const int32_t  kMaxInt32  =          2147483647;   // 2**31 - 1
-  const uint32_t kMaxUInt32 =          4294967295;   // 2**32 - 1
-  const int64_t  kMaxInt64  = 9223372036854775806;   // 2**63 - 2: see below
-  const int64_t  kSliceNone = kMaxInt64 + 1;         // for Slice::none()
-  const int64_t  kMaxLevels =                  48;
+const int8_t kMaxInt8 = 127;                    // 2**7  - 1
+const uint8_t kMaxUInt8 = 255;                  // 2**8  - 1
+const int32_t kMaxInt32 = 2147483647;           // 2**31 - 1
+const uint32_t kMaxUInt32 = 4294967295;         // 2**32 - 1
+const int64_t kMaxInt64 = 9223372036854775806;  // 2**63 - 2: see below
+const int64_t kSliceNone = kMaxInt64 + 1;       // for Slice::none()
+const int64_t kMaxLevels = 48;
```

Comment:
I liked the right-formatting of these numbers, but it's impressive that it exactly maintained the alignment of the comments (which is more important, anyway).